src/components/EditableTicket/EditableTicket.tsx

```diff
@@ -7,14 +7,18 @@ import Card from "../Card";
 export interface EditableTicketProps {
   value?: string;
   placeholder?: string;
-  onBlur?: (value: string) => void;
+  onChange?: (value: string) => void;
+  onEnter?: (value: string) => void;
+  onBlur?: () => void;
```

Comment:
Are these properties supposed to be optional? It looks like `<ColumnFooter />` calls this component with them, so the question is; is there any other place you need this component that might not have them?🤔

Reply:
It will be used in UI in Dialog to update ticket title. In that part blur won't be used. EditableTicket is more generic component and as all html component keep on... property optional, I would like to keep them optional in generic component.